casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
+    // <group>
+    static void setMemoryTotal(int memory);
```

Comment:
also casa only supports 64 bit linux, so in bindings you can also just use long or longlong which is the same on that platform